tutorials/intro/70_report.py

```diff
@@ -650,22 +647,6 @@
     )
 report.save("report_parse_folder_evoked.html", overwrite=True)
 
-# %%
-# If you want to actually *view* the noise covariance in the report, make sure
-# it is captured by the pattern passed to :meth:`~mne.Report.parse_folder`, and
-# also include a source for an :class:`~mne.Info` object (any of the
-# :class:`~mne.io.Raw`, :class:`~mne.Epochs` or :class:`~mne.Evoked`
-# :file:`.fif` files that contain subject data also contain the measurement
-# information and should work):
-
-pattern = "sample_audvis-cov.fif"
-info_fname = sample_dir / "sample_audvis-ave.fif"
-report = mne.Report(title="parse_folder example 5", info_fname=info_fname)
-report.parse_folder(
-    data_path, pattern=pattern, render_bem=False, n_time_points_evokeds=5
-)
-report.save("report_parse_folder_cov.html", overwrite=True)
```

Comment:
why removing this?

Reply:
There is already a section on noise cov viz above. The point is already made elsewhere that parse_folder is cool and usable. So what this adds is explanation of what I'd consider a rare enough use case to omit.

I feel like explaining so many options and configurations can actually be a detriment sometimes. This tutorial webpage is quite long ago I think condensing is useful. I was tempted to further roll add_raw/events/epochs/evoked/cov into a "sensor" block/report and add_bem/fwd/inv/stc into a "source" report rather than generating 9 separate reports for these for example. But that seemed a bit drastic and wouldn't speed things up anyway...

Reply:
👌